udata/templates/dataset/display.html

```diff
@@ -387,7 +389,9 @@ <h3 class="text-left">{{ _('Producer') }}</h3>
         <div class="row">
             <div class="col-sm-9 list-group resources-list smaller">
             {% for resource in dataset.community_resources %}
-                {% include theme('dataset/resource/list-item.html') %}
+                <div itemprop="hasPart">
```

Comment:
OK, je tested, the real error is the resource item template missing the `itemprop="distribution"` here: https://github.com/etalab/udata/blob/master/udata/templates/dataset/resource/list-item.html#L3

This will probable conflict with `hasPart` but should provide a richier integration